components/Footer/Footer.js

```diff
@@ -1,12 +1,14 @@
+import Link from "next/link";
+
 const Footer = () => {
 	return (
-		<footer className='py-6 bg-gray-100 text-gray-900'>
+		<footer className='py-6 dark:bg-gray-800 dark:text-gray-50'>
 			<div className='container px-6 mx-auto space-y-6 divide-y divide-gray-400 md:space-y-12 divide-opacity-50'>
 				<div className='grid grid-cols-12'>
 					<div className='pb-6 col-span-full md:pb-0 md:col-span-6'>
 						<a
```

Comment:
Use the `<Link>` tag instead since it is navigation between pages. Using an `<a>` is only used for external links